core/server/src/main/java/alluxio/worker/block/BlockMetadataManagerView.java

```diff
@@ -229,4 +229,15 @@ public BlockMeta getBlockMeta(long blockId) throws BlockDoesNotExistException {
       return null;
     }
   }
+
+  /**
+   * Return the byte size of the block.
+   *
+   * @param blockId id of the block
+   * @return block size
+   * @throws BlockDoesNotExistException if no {@link BlockMeta} for this block id is found
+   */
+  public long getBlockSizeIfExist(long blockId) throws BlockDoesNotExistException {
```

Comment:
@uronce-cc I also need `BlockMeta` in `onCommit`. If the signature of `onRemove` is `BlockMeta`, then I don't need to maintain block size of all blocks with another data structure.